### PR TITLE
New install page for admin + redirection

### PR DESCRIPTION
**Pull request by ildyria**
- add install step in the web installer screen.
- redirect to that page if an admin user does not exists
- add various securities.

### CODE REVIEW

File: app/Http/Controllers/Install/SetUpAdminController.php [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+	/**
+	 * Set up the admin user.
+	 * Called on POST request.
+	 *
+	 * @return View
+	 *
+	 * @throws FrameworkException
+	 */
+	public function view(SetUpAdminRequest $request): View
```
Comment (qwerty287) [inline app/Http/Controllers/Install/SetUpAdminController.php]: Don't call this `view`. The other method should be `view`, and this one something like `create` or `post`.

File: resources/views/install/setup-admin.blade.php [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+			<div class="buttons-container">
+				<button type="submit" class="button float-right" >
+					<i class="fa fa-check fa-fw" aria-hidden="true"></i>
+					Create admin account
+					<i class="fa fa-angle-double-right fa-fw" aria-hidden="true"></i>
+				</button>
+			</div>
+		</form>
+	@else
+		<strong>Admin account has been created.</strong>
```
Comment (qwerty287) [inline resources/views/install/setup-admin.blade.php]: Move to new blade template if it's successful.

File: app/Http/Controllers/Install/SetUpAdminController.php (hunk 1 above)
Comment (ildyria) [reply app/Http/Controllers/Install/SetUpAdminController.php]: Fixed [9ff238b](https://github.com/LycheeOrg/Lychee/pull/1660/commits/9ff238b976bd10c36d8dbc7f98e9d8696abd80d8).

File: resources/views/install/setup-admin.blade.php (hunk 2 above)
Comment (ildyria) [reply resources/views/install/setup-admin.blade.php]: Fixed [9ff238b](https://github.com/LycheeOrg/Lychee/pull/1660/commits/9ff238b976bd10c36d8dbc7f98e9d8696abd80d8).

**Review by qwerty287 — APPROVED** [review]
Looks good, but didn't test myself.

**Comment by ildyria** [comment]
> Looks good, but didn't test myself.

I did :)

**Comment by ildyria** [comment]
When no admin exists, we are redirected here:
image

When DB is not writable or for some other reason:
image

On success:
image

When trying to access again:
image

**Review by qwerty287 — APPROVED** [review]

**Pull request merged**